src/components/OverrideModal/OverrideModal.js

```diff
@@ -77,6 +83,25 @@ function OverrideModal(props) {
       : onOverride(omit(overrideItem, 'dueDate'));
   };
 
+  const renderPatronBlocks = patronBlocks.map(block => {
```

Comment:
Could you please clarify do we need change order here or limit blocks count? example below:
const blocks = take(orderBy(patronBlocks, ['metadata.updatedDate'], ['desc']), 3);

Reply:
Thanks much, I really missed that